### DISCUSSION TITLE
The resource "Discord Link" was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.

### DISCUSSION THREAD

**Discussion by nicebolt** [Papela/Nitrox-Cracked-Mod#22] [Q&A] [answered]
When I launched the Server, and I click the Multiplayer button, it shows this on my Commands,

12:15:35.175 > The resource https://discord.com/assets/452d7be36bf4b23241bd.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:15:35.176 > The resource https://discord.com/assets/8234e0a75aa9afb205bd.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:15:35.176 > The resource https://discord.com/assets/8bd8143eff37936894aa.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:15:35.177 > The resource https://discord.com/assets/f5b8aa3411dfc24ff2e6.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:15:35.177 > The resource https://discord.com/assets/b21c5111a12372139409.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:15:35.178 > The resource https://discord.com/assets/48a594e29497835802fe.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:16:03.457 > The resource https://discord.com/assets/452d7be36bf4b23241bd.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:16:03.458 > The resource https://discord.com/assets/8234e0a75aa9afb205bd.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:16:03.459 > The resource https://discord.com/assets/8bd8143eff37936894aa.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:16:03.459 > The resource https://discord.com/assets/f5b8aa3411dfc24ff2e6.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:16:03.460 > The resource https://discord.com/assets/b21c5111a12372139409.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:16:03.460 > The resource https://discord.com/assets/48a594e29497835802fe.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:16:20.231 > The resource https://discord.com/assets/452d7be36bf4b23241bd.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:16:20.232 > The resource https://discord.com/assets/8234e0a75aa9afb205bd.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:16:20.233 > The resource https://discord.com/assets/8bd8143eff37936894aa.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:16:20.233 > The resource https://discord.com/assets/f5b8aa3411dfc24ff2e6.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:16:20.234 > The resource https://discord.com/assets/b21c5111a12372139409.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
12:16:20.234 > The resource https://discord.com/assets/48a594e29497835802fe.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.

Can anyone help me with this?

**Comment by Papela** [comment]
What game version are u using?

**Reply by nicebolt** [reply, marked as answer]
I'm using v71288 right now, but nevermind, I already figured it out. So what I do is just click all the links, and it directs me to the browser and automatically downloads all these files. After I download all the files, it works for idk reason. I even didn't move these files into the game folder, but it works anyway.

**Reply by nicebolt** [reply]
Btw should I edit this thread? I mean to remove those link, just worried it affect for my laptop privacy or maybe nah?
